src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Right now this is a function-based component but according to the project guidelines, you are directed to use class-based components. Kindly convert this into a class component.